project/Build.scala

```diff
@@ -129,13 +129,14 @@ object DottyBuild extends Build {
     settings(publishing)
 
 
-  lazy val `dotty-interfaces` = project.in(file("interfaces")).
+  lazy val `dotty-interfaces`: Project = project.in(file("interfaces")).
     settings(sourceStructure).
+    dependsOn(`dotty-library`).
```

Comment:
We shouldn't do this. We need `binaryCompatible` to be on the classpath when compiling `dotty-interfaces` but it shouldn't be a runtime dependency.